### PR TITLE
feat: implement the initial iteration of front-end design

### PR DESCRIPTION
**Pull request by laurimiller**

### CODE REVIEW

File: api/_key.js [hunk 1]
```diff
@@ -2,36 +2,36 @@ import { strict as assert } from 'assert';
 import { readFileSync } from 'fs';
 
 const loadFromEnv = (varName) => {
-	const value = process.env[varName];
-	if (typeof value !== 'string') {
-		throw new Error(`Environment var "${varName}" is unset`);
-	}
-	const data = Buffer.from(value, 'base64').toString('utf8');
+    const value = process.env[varName];
```
Comment (Rauno56) [inline api/_key.js:5]: Please remove changes to this file

File: public/index.html [hunk 2]
```diff
@@ -1,18 +1,90 @@
...
+            </div>
+
+        </div>
+    </div>
 </body>
-</html>
+
+<script type="module" src="./index.js"></script>
+
+</html>
```
Comment (Rauno56) [inline public/index.html:90]: Add line breaks to the last lines for every file.

File: public/index.html [hunk 3]
```diff
@@ -1,18 +1,90 @@
...
-	  <label>Isikukood: <input id="idCode" type="text"></label>
-	  <br><br>
-	  <button type="submit">Kontrolli</button>
-	</form>
-	<p id="log"></p>
+    <div id="app">
+        <div id="left-background"></div>
+        <div id="left">
+            <form id="form" @submit.prevent="submit">
+                <p>Validate the climbing certifciate using the ID code</p>
```
Comment (Rauno56) [inline public/index.html:21]: Let's translate everything to Estonian before the merge? That way we can give it to the reception people in reality as well.

**Review by Rauno56 — COMMENTED** [review]
:+1: 

Do you think translating this to Estonian before would be too much work?

Also let's remove and ignore [public/.DS_Store](https://github.com/Rauno56/climbing-card-check/pull/2/files#diff-3ee769db5b814bc7f9bc3f8787f0d1f1483a0dc730faa6f37090506702717c45).

File: public/index.html [hunk 4]
```diff
@@ -1,18 +1,125 @@
...
+                    </div>
+                </div>
+                <!-- <p v-if="!showInstructions" class="warning">Validate the card information with the ID data of the climber</p> -->
+                <p v-if="!showInstructions" class="warning">Veendu, et ronija on isikuttõendava dokumendi omanik</p>
+                <div v-if="showInstructions" class="instructions">
+                    <div class="row">
+                        <div class="row-number-wrapper">
+                            <div class="row-number">1</div>
+                        </div>
+                        <p>Palu seinale ligipääsu soovijate isikuttõendavaid dokumente.</p>
```
Comment (Rauno56) [inline public/index.html:67]: "Isikut tõendav".

**Review by Rauno56 — CHANGES_REQUESTED** [review]

File: api/_key.js (hunk 1 above)
Comment (laurimiller) [reply api/_key.js:4]: Should be just changes from the eslint branch now

File: public/index.html (hunk 2 above)
Comment (laurimiller) [reply public/index.html]: Should be fixed

File: public/index.html (hunk 3 above)
Comment (laurimiller) [reply public/index.html]: All the existing text is translated now if im not mistaken

File: public/index.html (hunk 4 above)
Comment (laurimiller) [reply public/index.html]: Fixed

**Review by Rauno56 — APPROVED** [review]

**Pull request merged**